test/subset/data/Makefile.sources

```diff
@@ -16,6 +16,7 @@ TESTS = \
 	tests/layout.gpos2.tests \
 	tests/layout.gpos3.tests \
 	tests/layout.gpos4.tests \
+	tests/layout.gpos5.tests \
```

Comment:
Feel free to update respective meson.build list also, thanks :)

Reply:
Good catch, fixed.